docs/application.yaml

```diff
@@ -24,6 +24,20 @@ spec:
     # directory
     directory:
       recurse: true
+      jsonnet:
+        # A list of Jsonnet External Variables
+        extVars:
+        - key: foo
+          val: bar
```

Comment:
Despite being more verbose, we probably want to stick to the syntax:
```
name: foo
value: bar
```

The `name:` field in kubernetes is treated specialyl in kubernetes, because things like strategic merge patch have special understanding of updating a list of items, where `name` is used as the key. And it will be consistent with things like environment variables.

Reply:
Done.